### PR TITLE
Bug: Fix api augment module

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to update frequency-chain/api-augment to work in an esm package.

Part of #1116

# Discussion

- Updated depedencies
- Updated default node to v18
- Fix `make js` to always `npm pack`
- Clean up and apply a simple `package.json` to the esm and cjs folders with the right `type`

## Testing
- It's a pain.
- Best to wait until this merges and we cut 1.5.2
- You really want to?
- Ok.
- run make integration-test
- That should work (common-js usage)
- `touch ./js/api-augment/dist/yarn.lock`
- Switch to a folder with the `polkadotjs/apps` repo
- Search the repo for `https://github.com/polkadot-js/apps/pull/9243` and uncomment the lines about frequency
- `yarn`
-`yarn build` Should FAIL
- `yarn link ../frequency/js/api-augment/dist` (with your own path to frequency)
- Might need to move the change from the root package.json to the packages/apps-config package.json
- `yarn build` Should SUCCESS!

### CODE REVIEW

**Review by shannonwells — APPROVED** [review]
WOMM

File: scripts/generate_js_definitions.sh [hunk 1]
```diff
@@ -41,8 +41,8 @@ else
...
     npm run fetch:local
     npm run build
-
-    # Generate the new packed tgz
-    cd dist
-    npm pack
 fi
+
+# Generate the new packed tgz
+cd dist
```
[the remark this answers is not in the document]
Comment (wilwade) [reply scripts/generate_js_definitions.sh:47]: Right. Both trigger the build, so both should trigger the pack.

File: integration-tests/.load-test.mocharc.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "exit": true,
+    "extension": ["ts"],
+    "parallel": false,
+    "require":  ["scaffolding/rootHooks.ts", "scaffolding/extrinsicHelpers.ts"],
+    "loader": "ts-node/esm",
+    "spec": ["./load-tests/**/*.test.ts"],
+    "timeout": 60_000,
```
Comment (aramikm) [inline integration-tests/.load-test.mocharc.json:8]: Does json format supports numbers with underscore? Seems Github viewer doesn't seem to like it.

**Review by aramikm — APPROVED** [review]
Looks good to me!

File: integration-tests/.load-test.mocharc.json (hunk 2 above)
Comment (wilwade) [reply integration-tests/.load-test.mocharc.json:8]: I think it works due to being processed by typescript

Comment (wilwade) [reply integration-tests/.load-test.mocharc.json:8]: I'll remove it in a future PR

**Pull request merged**